modules/islandora_binary_object_zip_importer/includes/importer.inc

```diff
@@ -79,7 +79,7 @@ class BinaryObjectZipBatchImporter extends ZipBatchImporter {
    * @see IslandoraBatchImporterInterface::getBatchInfo()
    */
   public static function getBatchInfo(array &$form_state) {
-    $file = $form_state['values']['file'];
+    $file = is_object($form_state['values']['file']) ? $form_state['values']['file'] : file_load($form_state['values']['file']);
```

Comment:
... Is the `is_object()` really necessary here?... As in, it should _always_ have to be `file_load()`'d... And in fact does in [the upstream code off which this is based](https://github.com/Islandora/islandora_importer/blob/7.x/modules/zip_importer/includes/importer.inc#L119).